src/main/java/xyz/aadev/generitech/common/blocks/power/BlockCables.java

```diff
@@ -1,6 +1,13 @@
 package xyz.aadev.generitech.common.blocks.power;
 
 
+import ibxm.Player;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'ibxm.Player'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)